evennia/utils/containers.py

```diff
@@ -208,18 +208,8 @@ def load_data(self):
         if self.typeclass_storage is None:
```

Comment:
Remember to remove 205-206 and 22. We no longer need `_BASE_SCRIPT_TYPECLASS` I added.

Reply:
Good catch, thanks